components/elm/cime_config/testdefs/testmods_dirs/elm/lnd_rof_2way/user_nl_elm

```diff
@@ -0,0 +1 @@
+ use_lnd_rof_two_way = .true.
```

Comment:
@bishtgautam and I have discussed about how to make sure those flags from rof and lnd to be consistent, but we didn't come up with a good solution. The land river two-way coupling will not be implemented correctly if one flag is turned on but the other remains off. 

Do you know how to make sure one flag from rof is consistent with another flag from lnd? It will be great if we can let E3SM crash and report error if they are not consistent.

Reply:
No I don't. It would be nice to have something like that.